Sources/PrimerSDK/Classes/User Interface/UIUtils.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UIUtils.swift
+//  PrimerSDK
+//
+//  Created by Dario Carlomagno on 14/02/22.
+//  Copyright © 2022 Primer API ltd, Inc. All rights reserved.
+//
+
+import Foundation
+
+///
+/// Utility Typealias that returns a Void callback
+/// For implementation example check PrimerButton.swift
+///
+public typealias Action = () -> Void
```

Comment:
Otherwise `PrimerButton:68` won't be accessible from whoever would like to implement the Button with the callback even: `public var onPressed: Action?`